gui/elm/Window/Updater.elm

```diff
@@ -77,7 +78,8 @@ view helpers model =
                 ]
 
             ( Nothing, Just progress ) ->
-                [ h1 [] [ text (helpers.t "Updater Downloading") ]
+                [ div [] [ figure [] [ Icons.logo ] ]
```

Comment:
I don't want to have `Icons.logo` to include other markup like `figure` as in `Icons.cozyBig`.